BackofficeBundle/EventSubscriber/DataTransformer/HtmlStringTransformer.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer;
+
+/**
+ * Class HtmlStringTransformer
+ */
+class HtmlStringTransformer implements ValueTransformerInterface
```

Comment:
As you directly return the value in the transformerManager, this transformer has become useless, hasn't it ?